index.html

```diff
@@ -0,0 +1,139 @@
+                    <div class="wrapper w960p flex-container flex-container-v">
+                        <div class="separator w10"></div>
+                        <h1 class="xbiggest flex-item-center txtcenter mtxl">FAQs</h1>
+                        
+                        <div class="features-container mtxl">
+                            <div class="grid-2 has-gutter">
+                                <div class="Media mbl">
+                                    <div class="Media-body big prm"><strong>Do you have a chat I can join? </strong>Yes, we do. For general discussions, please join this <a href ="https://t.me/joinchat/G60ltQv5CCEQ94BZ5yWQbg">Telegram chat.</a> For updates on test releases, please join this <a href ="https://t.me/AndrOBD_dev">Telegram chat.</a></div>
+                                </div>
+                                <div class="Media mbl">
+                                    <div class="Media-body big prm"><strong>How can I report a bug?</strong> You can use <a href="https://github.com/fr3ts0n/AndrOBD/issues">GitHub issues</a>. to report any issues.</div>
+                                </div>
+                            </div>
+                            
+                            <div class="grid-2 has-gutter">
+                                <div class="Media mbl">
+                                    <div class="Media-body big prm"><strong>How can I support the project?</strong> One area you can help with is maintaining the project on <a href ="https://github.com/fr3ts0n/AndrOBD">GitHub.</a> If that's not your type of thing, you can fincailly support the project by donating on <a href ="https://www.paypal.me/fr3ts0n">Paypal</a> or Bitcoin (19UApzsc5eDJ5VNDNYCA1bpszPnkcpWeFP). Thanks to John Zimmerer, Martin Bourdoiseau, Jeffrey O'Connell & Christoph Schmid for their support.</div>
+                                </div>
+                                <div class="Media mbl">
+                                    
+                                    <div class="Media-body big prm"><strong>What plugins are available?</strong> You can learn more about plugins <a href="https://github.com/fr3ts0n/AndrOBD-Plugin">here</a></div>
+                                </div>
+                            </div>
+                            
+                            
+                            
+                        </div>
+                    </div>
+                </section>
+                
+               
+                <footer class="footer-section mtxl">
+                    <div class="wrapper w960p flex-container flex-container-v txtcenter">
+                         
+                        <div class="h5-like pbs">Built by fr3ts0n • Hosted on <a href="https://github.com/fr3ts0n/AndrOBD">GitHub</a></div>
+                        <div class="h5-like pbl"><a href="https://github.com/fr3ts0n/AndrOBD-Plugin/blob/master/LICENSE"> Licensed under GNU General Public License v3.0</a></div>
```

Comment:
Link is formally incorrect, since it links to licence of plugin repo

Reply:
Sorry! Let me fix this

Reply:
I've fixed this